ValidatorPanel.php

```diff
@@ -853,6 +858,13 @@ public function stopBuffering()
 		if (ob_get_level()) ob_end_flush();
 	}
 
+	/**
+	 * Stop validator.
+	 */
+	public static function stopValidate()
```

Comment:
Stop validate IMHO means to stop something that is already running. What about `disableValidation()`?